File: Algorithms/BubbleSort.py

def bubble_sort(list_to_be_sorted):
	for number_1 in range(len(list_to_be_sorted)):
		for number_2 in range(len(list_to_be_sorted)):
			if list_to_be_sorted[number_1] < list_to_be_sorted[number_2]:
				list_to_be_sorted[number_1], list_to_be_sorted[number_2] = list_to_be_sorted[number_2], list_to_be_sorted[number_1]
				
	return list_to_be_sorted

if __name__ == "__main__":
    list_to_be_sorted = [5, 9, 3, 1, 2, 8, 4, 7, 6]
    print("Not sorted list: ")
    print(list_to_be_sorted)
    print()
    sorted_list = bubble_sort(list_to_be_sorted)
    print("Sorted list: ")
    print(sorted_list)

File: Algorithms/BubbleSort.java

public class Main
{
	
    static void bubbleSort(int[] listToBeSorted) {
        for (int number1 = 0; number1 < listToBeSorted.length; number1++) {
            for (int number2 = 0; number2 < listToBeSorted.length; number2++) {
                if (listToBeSorted[number1] < listToBeSorted[number2]) {
                    int temporaryValue = listToBeSorted[number1];
                    listToBeSorted[number1] = listToBeSorted[number2];
                    listToBeSorted[number2] = temporaryValue;
                }
            }
        }
        
        for (int number = 0; number < listToBeSorted.length; number++) {
            System.out.print(listToBeSorted[number] + ", ");
        }
    }
    
	public static void main(String[] args) {
		int[] listToBeSorted = {5, 9, 3, 1, 2, 8, 4, 7, 6};
		System.out.println("Not sorted list: ");
		
		for (int number = 0; number < listToBeSorted.length; number++) {
            System.out.print(listToBeSorted[number] + ", ");
        }
		
		System.out.println();
		System.out.println("Sorted list: ");
		bubbleSort(listToBeSorted);
	}
}
